linting-configs/.tflint.hcl

```diff
@@ -9,7 +9,7 @@
 
 config {
   # Enables module inspection.
-  module = true
+  call_module_type = "local"
```

Comment:
Consider changing the right-side value to "all" instead of "local" as shown in this change https://github.com/sassoftware/viya4-iac-k8s/compare/main...staging#diff-691eaaf54fb2546bf9ce095e46dd4513050e18c9524bfe1258299f39e435880fR11